changes/2075.fixed

```diff
@@ -0,0 +1 @@
+Fixed white screen flash in dark mode. Updated dark mode theme to be less...dark. 
```

Comment:
```suggestion
Fixed white screen flash in dark mode.
```

You can add a separate `2493.changed` file for `Updated dark mode theme to be less...dark.`

Reply:
Done